app/services/flow/flows/provider_start.rb

```diff
@@ -130,7 +130,7 @@ class ProviderStart < FlowSteps
         },
         bank_statements: {
           path: ->(application) { urls.providers_legal_aid_application_bank_statements_path(application) },
-          forward: :identify_types_of_incomes,
+          forward: :employment_incomes,
```

Comment:
I think this may need to be more like the client_completed_means step

```
          forward: lambda do |application|
            status = HMRC::StatusAnalyzer.call(application)
            case status
            when :hmrc_multiple_employments, :no_hmrc_data
              :full_employment_details
            when :hmrc_single_employment, :unexpected_employment_data
              :employment_incomes
            when :employed_journey_not_enabled, :provider_not_enabled_for_employed_journey, :applicant_not_employed
              application.income_types? ? :income_summary : :no_income_summaries
            else
              raise "Unexpected hmrc status #{status.inspect}"
            end
          end,
```

but modified to account to for the new journey

Reply:
Consideration of how long HMRC response will take - is it long enough to actually get a response?